docs/store/datastore.md

```diff
@@ -179,6 +179,7 @@ The equivalent to this parameter in environment authentication is "AZURE_STORAGE
 Credential authentication:
 - `credential` &mdash; TokenCredential or SAS token. The credentials with which to authenticate.
 This variable is sensitive information and is kept confidential.
+- `bucket` &mdash; A string representing the bucket. When specified, it is automatically appended to the object path, and thus, it should not be manually included in the target path by the user. 
```

Comment:
```suggestion
- `bucket` &mdash; A string representing the bucket. When specified, it is automatically prepended to the object path, and thus, it should not be manually included in the target path by the user. 
```
(all instances)

Reply:
Also, need to add the fact that the user needs to specify this, and will be required to do so from 1.9.0 onward.